style.css

```diff
@@ -246,11 +297,34 @@ a {
 
 .beatEmphasisText {
   /* need more practice with vmin concept */
+  padding-top: 10px;
   font-size: 2vmin;
   min-width: 5px;
   background-color: transparent;
   text-align: center;
-  font-family: "Courier New", Courier, monospace;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+/* input[type="checkbox"] means only that input type is styled */
+/* this makes the checkbox itself disappear, I think */
+input[type="checkbox"] {
+  display: none;
+}
+
+/* label style when NOT checked */
+input[type="checkbox"] + label {

```

Comment:
The position of the volume icon within the button looks a little wonky. :-) Maybe because of the white space that is already built into the icon? You might need to play around with it a bit.